bpmsuite/bpmsuite70-businesscentral-monitoring-with-smartrouter.json

```diff
@@ -49,173 +63,109 @@
+            "description": "KIE server controller username (adds the user)",
             "name": "KIE_SERVER_CONTROLLER_USER",
             "value": "controllerUser",
             "required": false
         },
         {
             "displayName": "KIE Server Controller Password",
-            "description": "KIE server controller password (Sets the org.kie.server.controller.pwd system property)",
+            "description": "KIE server controller password (adds the user)",
             "name": "KIE_SERVER_CONTROLLER_PWD",
-            "from": "[a-zA-Z]{6}[0-9]{1}!",
-            "generate": "expression",
+            "value": "controllerPassword",
             "required": false
         },
         {
             "displayName": "KIE Server User",
-            "description": "KIE execution server username (Sets the org.kie.server.user system property)",
+            "description": "KIE execution server username (Sets the org.kie.server.user system property, used by the embedded controller)",
             "name": "KIE_SERVER_USER",
             "value": "executionUser",
             "required": false
         },
         {
             "displayName": "KIE Server Password",
-            "description": "KIE execution server password (Sets the org.kie.server.pwd system property)",
+            "description": "KIE execution server password (Sets the org.kie.server.pwd system property, used by the embedded controller)",
             "name": "KIE_SERVER_PWD",
-            "from": "[a-zA-Z]{6}[0-9]{1}!",
-            "generate": "expression",
-            "required": false
-        },
-        {
-            "displayName": "Application Name",
-            "description": "The name for the application.",
-            "name": "APPLICATION_NAME",
-            "value": "myapp",
-            "required": true
-        },
-        {
-            "displayName": "Business Central Monitoring Custom http Route Hostname",
-            "description": "Custom hostname for http service route.  Leave blank for default hostname, e.g.: <application-name>-buscentrmon-<project>.<default-domain-suffix>",
-            "name": "BUSINESS_CENTRAL_MONITORING_HOSTNAME_HTTP",
-            "value": "",
+            "value": "executionPassword",
             "required": false
         },
         {
-            "displayName": "Busines Central Monitoring Custom https Route Hostname",
-            "description": "Custom hostname for https service route.  Leave blank for default hostname, e.g.: secure-<application-name>-buscentrmon-<project>.<default-domain-suffix>",
-            "name": "BUSINESS_CENTRAL_MONITORING_HOSTNAME_HTTPS",
-            "value": "",
+            "displayName": "Smart Router listening port",
+            "description": "Port in which the smart router server listens (router property org.kie.server.router.port)",
+            "name": "KIE_ROUTER_PORT",
```

Comment:
Recently Smart router variables have been changed, see renamed variables in  [1]. KIE_ROUTER_PORT was renamed to KIE_SERVER_ROUTER_PORT.

[1] https://github.com/jboss-container-images/jboss-bpmsuite-7-openshift-image/blob/bpmsuite70-dev/smartrouter/image.yaml

Reply:
Correct. These variables were misnamed originally.

Reply:
Not sure what went wrong here. My intention was not to touch this. I only wanted to remove the DB related configs. Will need to review everything again.